frontend/src/store/queries/comparisons/index.ts

```diff
@@ -53,6 +53,21 @@ export const comparisonsApi = api.injectEndpoints({
       query: () => `${API.COMPARISONS}/general`,
       providesTags: ['Comparisons'],
     }),
+    getComparisonCars: builder.query<ComparisonInfo[], void>({
+      query: () => ({
+        url: API.COMPARISONS,
+        method: 'GET',
+        providesTags: ['Comparisons'],
```

Comment:
I pulled your code to my branch and noticed some mistake. In this code you need added "providesTags" field after query method.